net/golioth/Kconfig

```diff
@@ -39,6 +39,12 @@ config GOLIOTH_FW
 	  Enable Golioth firmware management, which allows to ask for newest
 	  desired firmware and issue firmware download process.
 
+config GOLIOTH_RPC
+	bool "Remote Procedure Call"
+	default n
```

Comment:
As in upstream Zephyr project, please remove `default n` as all options are disabled by default.